app/views/providers/legal_aid_applications/_legal_aid_applications.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <table class="govuk-table">
+      <caption class="govuk-table__caption">Current applications</caption>
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Client's name</th>
```

Comment:
Should we store these in the locale file?

Reply:
Good point!!